docs/developer-tools/postman.md

```diff
@@ -14,7 +14,7 @@ You need to have `appId` and `appSecret` to use authenticate before you can run
 ## How to use Symbl postman collection?
 
 <p align="left">
-<iframe width="850" height="550" src="https://www.loom.com/embed/2fcc10b6652347d09632172153748d57" frameborder="0" allowfullscreen mozallowfullscreen ></iframe>
+<iframe width="850" height="550" src="https://www.loom.com/share/407fff46c55344cdae176f10ddb5f3c0" frameborder="0" allowfullscreen mozallowfullscreen ></iframe>
```

Comment:
I believe "/share/" in the URl should switch back to "/embed/" -- I tested the version with "embed" in the browser and it works as expected